features/admin.console-settings.v1/components/console-administrators/console-administrators.tsx

```diff
@@ -186,8 +186,12 @@ const ConsoleAdministrators: FunctionComponent<ConsoleAdministratorsInterface> =
                     value={ activeAdministratorGroup }
                     onChange={ (_: ChangeEvent<HTMLInputElement>, value: string) => setActiveAdministratorGroup(value) }
                 >
-                    <FormControlLabel value="administrators" control={ <Radio /> } label="Administrators" />
-                    <FormControlLabel value="privilegedUsers" control={ <Radio /> } label="Privileged Users" />
+                    <FormControlLabel value="administrators" control={ <Radio /> } label="Asgardeo" />
```

Comment:
Lets add `i18n` placeholders.
Check the other Radio as well.

Reply:
Fixed with a6a5a27